### DISCUSSION TITLE
项目打包时怎样配置运行环境？

### DISCUSSION THREAD

**Discussion by yesay** [empjs/emp#310] [Q&A]
vue2 项目执行打包命令 emp build 时，怎样选择不同的环境？
我这是一起显示的 production

![2022-09-01_171534](https://user-images.githubusercontent.com/3607509/187878552-e0958031-735f-4a79-b1fc-c67a7bf97110.png)

![2022-09-01_171346](https://user-images.githubusercontent.com/3607509/187878231-eaa8b4e3-c78f-4c9a-bea3-6cc233e6acbb.png)
![2022-09-01_171636](https://user-images.githubusercontent.com/3607509/187878740-c6bb8072-cc99-463a-8a46-21287c356f6a.png)

**Comment by ckken** [comment]
![image](https://user-images.githubusercontent.com/3890513/188044622-2e63cb07-6b49-4b34-8d79-d83bbd77f335.png)

**Comment by ckken** [comment]
文档有介绍

**Comment by yesay** [comment]
项目中需要创建 .env.dev     .env.test      .env.prod    的文件吗？
